src/vscode-dts/vscode.d.ts

```diff
@@ -11183,6 +11183,10 @@ declare module 'vscode' {
 		/**
 		 * A human-readable string which will be used to describe the
 		 * operation.
+		 *
+		 * It would always be used as a prefix in later progress reports once given.
```

Comment:
Not a fan of such detailed messages because there is no guarantee that we won't change the UI. We don't want extensions to rely on this and craft titles that don't make sense when not rendered like this anymore

Reply:
Sure, that makes sense with UI evolving.